src/components.ts

```diff
@@ -203,6 +211,11 @@ export async function initComponents(): Promise<AppComponents> {
   })
   const communityOwners = createCommunityOwnersComponent({ catalystClient })
   const communityEvents = await createCommunityEventsComponent({ config, logs, fetcher, redis })
+
+  // AI Compliance components
+  const aiCompliance = await createAIComplianceComponent({ config, logs })
```

Comment:
Maybe we should only expose this component in higher environments (prevent execution of it in Zone, to prevent wasting tokens and instead of this component use a dummy one for dev), what do you think?